src/EnergyPlus/WindowAC.cc

```diff
@@ -1011,7 +1011,7 @@ namespace WindowAC {
                 zoneEqSizing.SizingMethod(SizingMethod) = CapSizingMethod;
                 if (CapSizingMethod == CoolingDesignCapacity || CapSizingMethod == CapacityPerFloorArea ||
                     CapSizingMethod == FractionOfAutosizedCoolingCapacity) {
-                    if (CapSizingMethod == HeatingDesignCapacity) {
+                    if (CapSizingMethod == CoolingDesignCapacity) {
```

Comment:
This was an existing logic error that I noticed

Reply:
Good catch. Without example files or unit tests these are hard to find.